.github/workflows/versioned-security-agent-pre.yml

```diff
@@ -0,0 +1,53 @@
+# Daily workflow to run versioned tests with security agent enabled
+# This also can be used to run versioned tests with a newer versioned of
+# security agent since we pin it.
+
+name: Versioned Tests w/ Security Agent-pre
+
+on:
```

Comment:
can you do the inverse of this @sumitsuthar? clone our repo into yours? that way you can manage failures in your own GH project

Reply:
This is only for our own forked repo. This will not merged with node-newrelic repo. Mistakenly PR was opened